.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,53 @@
+name: 🐞 Bug report
+description: Create a report to help us improve.
+labels: [bug]
```

Comment:
This adds the label bug to each bug report to enable simple filtering for bugs or feature request.